### ISSUE TITLE
During model training, no model file is output.

### ISSUE THREAD

**Issue by Ctrl-cv-engineer** [myshell-ai/MeloTTS#306]
Following the tutorial steps to train the model, there is no model file output. Below are the config and train.log files.

config：
`{
  "train": {
    "log_interval": 200,
    "eval_interval": 5,
    "seed": 52,
    "epochs": 10,
    "learning_rate": 0.0003,
    "betas": [
      0.8,
      0.99
    ],
    "eps": 1e-09,
    "batch_size": 2,
    "fp16_run": false,
    "lr_decay": 0.999875,
    "segment_size": 16384,
    "init_lr_ratio": 1,
    "warmup_epochs": 0,
    "c_mel": 45,
    "c_kl": 1.0,
    "skip_optimizer": true
  },
  "data": {
    "training_files": "/root/autodl-tmp/train.list",
    "validation_files": "/root/autodl-tmp/val.list",
    "max_wav_value": 32768.0,
    "sampling_rate": 44100,
    "filter_length": 2048,
    "hop_length": 512,
    "win_length": 2048,
    "n_mel_channels": 128,
    "mel_fmin": 0.0,
    "mel_fmax": null,
    "add_blank": true,
    "n_speakers": 1,
    "cleaned_text": true,
    "spk2id": {
      "lucy": 0
    }
  },
  "model": {
    "use_spk_conditioned_encoder": true,
    "use_noise_scaled_mas": true,
    "use_mel_posterior_encoder": false,
    "use_duration_discriminator": true,
    "inter_channels": 192,
    "hidden_channels": 192,
    "filter_channels": 768,
    "n_heads": 2,
    "n_layers": 6,
    "n_layers_trans_flow": 3,
    "kernel_size": 3,
    "p_dropout": 0.1,
    "resblock": "1",
    "resblock_kernel_sizes": [
      3,
      7,
      11
    ],
    "resblock_dilation_sizes": [
      [
        1,
        3,
        5
      ],
      [
        1,
        3,
        5
      ],
      [
        1,
        3,
        5
      ]
    ],
    "upsample_rates": [
      8,
      8,
      2,
      2,
      2
    ],
    "upsample_initial_channel": 512,
    "upsample_kernel_sizes": [
      16,
      16,
      8,
      2,
      2
    ],
    "n_layers_q": 3,
    "use_spectral_norm": false,
    "gin_channels": 256
  },
  "num_languages": 8,
  "num_tones": 16,
  "symbols": [
    "_",
    "\"",
    "(",
    ")",
    "*",
    "/",
    ":",
    "AA",
    "E",
    "EE",
    "En",
    "N",
    "OO",
    "Q",
    "V",
    "[",
    "\\",
    "]",
    "^",
    "a",
    "a:",
    "aa",
    "ae",
    "ah",
    "ai",
    "an",
    "ang",
    "ao",
    "aw",
    "ay",
    "b",
    "by",
    "c",
    "ch",
    "d",
    "dh",
    "dy",
    "e",
    "e:",
    "eh",
    "ei",
    "en",
    "eng",
    "er",
    "ey",
    "f",
    "g",
    "gy",
    "h",
    "hh",
    "hy",
    "i",
    "i0",
    "i:",
    "ia",
    "ian",
    "iang",
    "iao",
    "ie",
    "ih",
    "in",
    "ing",
    "iong",
    "ir",
    "iu",
    "iy",
    "j",
    "jh",
    "k",
    "ky",
    "l",
    "m",
    "my",
    "n",
    "ng",
    "ny",
    "o",
    "o:",
    "ong",
    "ou",
    "ow",
    "oy",
    "p",
    "py",
    "q",
    "r",
    "ry",
    "s",
    "sh",
    "t",
    "th",
    "ts",
    "ty",
    "u",
    "u:",
    "ua",
    "uai",
    "uan",
    "uang",
    "uh",
    "ui",
    "un",
    "uo",
    "uw",
    "v",
    "van",
    "ve",
    "vn",
    "w",
    "x",
    "y",
    "z",
    "zh",
    "zy",
    "~",
    "æ",
    "ç",
    "ð",
    "ø",
    "ŋ",
    "œ",
    "ɐ",
    "ɑ",
    "ɒ",
    "ɔ",
    "ɕ",
    "ə",
    "ɛ",
    "ɜ",
    "ɡ",
    "ɣ",
    "ɥ",
    "ɦ",
    "ɪ",
    "ɫ",
    "ɬ",
    "ɭ",
    "ɯ",
    "ɲ",
    "ɵ",
    "ɸ",
    "ɹ",
    "ɾ",
    "ʁ",
    "ʃ",
    "ʊ",
    "ʌ",
    "ʎ",
    "ʏ",
    "ʑ",
    "ʒ",
    "ʝ",
    "ʲ",
    "ˈ",
    "ˌ",
    "ː",
    "̃",
    "̩",
    "β",
    "θ",
    "ᄀ",
    "ᄁ",
    "ᄂ",
    "ᄃ",
    "ᄄ",
    "ᄅ",
    "ᄆ",
    "ᄇ",
    "ᄈ",
    "ᄉ",
    "ᄊ",
    "ᄋ",
    "ᄌ",
    "ᄍ",
    "ᄎ",
    "ᄏ",
    "ᄐ",
    "ᄑ",
    "ᄒ",
    "ᅡ",
    "ᅢ",
    "ᅣ",
    "ᅤ",
    "ᅥ",
    "ᅦ",
    "ᅧ",
    "ᅨ",
    "ᅩ",
    "ᅪ",
    "ᅫ",
    "ᅬ",
    "ᅭ",
    "ᅮ",
    "ᅯ",
    "ᅰ",
    "ᅱ",
    "ᅲ",
    "ᅳ",
    "ᅴ",
    "ᅵ",
    "ᆨ",
    "ᆫ",
    "ᆮ",
    "ᆯ",
    "ᆷ",
    "ᆸ",
    "ᆼ",
    "ㄸ",
    "!",
    "?",
    "…",
    ",",
    ".",
    "'",
    "-",
    "¿",
    "¡",
    "SP",
    "UNK"
  ]
}`

train.log:
`2025-12-02 13:34:44,129	autodl-tmp	INFO	{'train': {'log_interval': 200, 'eval_interval': 5, 'seed': 52, 'epochs': 10, 'learning_rate': 0.0003, 'betas': [0.8, 0.99], 'eps': 1e-09, 'batch_size': 2, 'fp16_run': False, 'lr_decay': 0.999875, 'segment_size': 16384, 'init_lr_ratio': 1, 'warmup_epochs': 0, 'c_mel': 45, 'c_kl': 1.0, 'skip_optimizer': True}, 'data': {'training_files': '/root/autodl-tmp/train.list', 'validation_files': '/root/autodl-tmp/val.list', 'max_wav_value': 32768.0, 'sampling_rate': 44100, 'filter_length': 2048, 'hop_length': 512, 'win_length': 2048, 'n_mel_channels': 128, 'mel_fmin': 0.0, 'mel_fmax': None, 'add_blank': True, 'n_speakers': 1, 'cleaned_text': True, 'spk2id': {'lucy': 0}}, 'model': {'use_spk_conditioned_encoder': True, 'use_noise_scaled_mas': True, 'use_mel_posterior_encoder': False, 'use_duration_discriminator': True, 'inter_channels': 192, 'hidden_channels': 192, 'filter_channels': 768, 'n_heads': 2, 'n_layers': 6, 'n_layers_trans_flow': 3, 'kernel_size': 3, 'p_dropout': 0.1, 'resblock': '1', 'resblock_kernel_sizes': [3, 7, 11], 'resblock_dilation_sizes': [[1, 3, 5], [1, 3, 5], [1, 3, 5]], 'upsample_rates': [8, 8, 2, 2, 2], 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 8, 2, 2], 'n_layers_q': 3, 'use_spectral_norm': False, 'gin_channels': 256}, 'num_languages': 8, 'num_tones': 16, 'symbols': ['_', '"', '(', ')', '*', '/', ':', 'AA', 'E', 'EE', 'En', 'N', 'OO', 'Q', 'V', '[', '\\', ']', '^', 'a', 'a:', 'aa', 'ae', 'ah', 'ai', 'an', 'ang', 'ao', 'aw', 'ay', 'b', 'by', 'c', 'ch', 'd', 'dh', 'dy', 'e', 'e:', 'eh', 'ei', 'en', 'eng', 'er', 'ey', 'f', 'g', 'gy', 'h', 'hh', 'hy', 'i', 'i0', 'i:', 'ia', 'ian', 'iang', 'iao', 'ie', 'ih', 'in', 'ing', 'iong', 'ir', 'iu', 'iy', 'j', 'jh', 'k', 'ky', 'l', 'm', 'my', 'n', 'ng', 'ny', 'o', 'o:', 'ong', 'ou', 'ow', 'oy', 'p', 'py', 'q', 'r', 'ry', 's', 'sh', 't', 'th', 'ts', 'ty', 'u', 'u:', 'ua', 'uai', 'uan', 'uang', 'uh', 'ui', 'un', 'uo', 'uw', 'v', 'van', 've', 'vn', 'w', 'x', 'y', 'z', 'zh', 'zy', '~', 'æ', 'ç', 'ð', 'ø', 'ŋ', 'œ', 'ɐ', 'ɑ', 'ɒ', 'ɔ', 'ɕ', 'ə', 'ɛ', 'ɜ', 'ɡ', 'ɣ', 'ɥ', 'ɦ', 'ɪ', 'ɫ', 'ɬ', 'ɭ', 'ɯ', 'ɲ', 'ɵ', 'ɸ', 'ɹ', 'ɾ', 'ʁ', 'ʃ', 'ʊ', 'ʌ', 'ʎ', 'ʏ', 'ʑ', 'ʒ', 'ʝ', 'ʲ', 'ˈ', 'ˌ', 'ː', '̃', '̩', 'β', 'θ', 'ᄀ', 'ᄁ', 'ᄂ', 'ᄃ', 'ᄄ', 'ᄅ', 'ᄆ', 'ᄇ', 'ᄈ', 'ᄉ', 'ᄊ', 'ᄋ', 'ᄌ', 'ᄍ', 'ᄎ', 'ᄏ', 'ᄐ', 'ᄑ', 'ᄒ', 'ᅡ', 'ᅢ', 'ᅣ', 'ᅤ', 'ᅥ', 'ᅦ', 'ᅧ', 'ᅨ', 'ᅩ', 'ᅪ', 'ᅫ', 'ᅬ', 'ᅭ', 'ᅮ', 'ᅯ', 'ᅰ', 'ᅱ', 'ᅲ', 'ᅳ', 'ᅴ', 'ᅵ', 'ᆨ', 'ᆫ', 'ᆮ', 'ᆯ', 'ᆷ', 'ᆸ', 'ᆼ', 'ㄸ', '!', '?', '…', ',', '.', "'", '-', '¿', '¡', 'SP', 'UNK'], 'model_dir': './logs/autodl-tmp', 'pretrain_G': None, 'pretrain_D': None, 'pretrain_dur': None, 'port': 10000}
2025-12-02 13:34:44,130	autodl-tmp	WARNING	/root/MeloTTS/melo is not a git repository, therefore hash value comparison will be ignored.
2025-12-02 13:34:49,494	autodl-tmp	ERROR	enc_p.language_emb.weight is not in the checkpoint
2025-12-02 13:34:49,495	autodl-tmp	ERROR	emb_g.weight is not in the checkpoint
2025-12-02 13:34:49,548	autodl-tmp	INFO	Loaded checkpoint '/root/.cache/cached_path/73ad3d5a37c82356ed81630b0a435b4b376ca49523854fe2b8302609fd71c193.133b77b9d9162e348486a0a0778fa47d726930e3ec12ea5e2684c0c919743a65' (iteration 0)
2025-12-02 13:34:49,681	autodl-tmp	INFO	Loaded checkpoint '/root/.cache/cached_path/c3d3c787a8711093a79ee95f091a35de75e527b6e8e28424ad7010f6e86cce58.e5f88bb1eca17c37beb511b15a932e84fdc8b66d8a8d5c5075334650425954f2' (iteration 0)
2025-12-02 13:34:49,698	autodl-tmp	INFO	Loaded checkpoint '/root/.cache/cached_path/c7b373ab8939eb672a985a802d21420534ca0cd43fa4aecf4fa6088a569ee2a1.ce7a8153914d9727ebc28e4b4e3d31eed35aa0b4e3d125eb54e6f8363968dd7a' (iteration 0)
2025-12-02 13:34:56,274	autodl-tmp	INFO	Train Epoch: 1 [0%]
2025-12-02 13:34:56,274	autodl-tmp	INFO	[2.866032600402832, 1.5802992582321167, 3.37438702583313, 15.872100830078125, 5.847896575927734, 7.971815586090088, 0, 0.0003]
2025-12-02 13:34:58,526	autodl-tmp	INFO	Train Epoch: 2 [0%]
2025-12-02 13:34:58,527	autodl-tmp	INFO	[2.643841505050659, 4.211231231689453, 26.638912200927734, 33.83002471923828, 3.579105854034424, 6.094124794006348, 0, 0.0002999625]
2025-12-02 13:35:00,569	autodl-tmp	INFO	Train Epoch: 3 [0%]
2025-12-02 13:35:00,569	autodl-tmp	INFO	[3.0783004760742188, 2.216205358505249, 4.120388507843018, 19.018903732299805, 3.9273838996887207, 5.636021614074707, 0, 0.0002999250046875]
2025-12-02 13:35:03,857	autodl-tmp	INFO	Train Epoch: 4 [0%]
2025-12-02 13:35:03,858	autodl-tmp	INFO	[3.100656747817993, 3.4044301509857178, 18.928205490112305, 27.78141975402832, 3.725606679916382, 5.340543746948242, 0, 0.00029988751406191406]
2025-12-02 13:35:06,026	autodl-tmp	INFO	Train Epoch: 5 [0%]
2025-12-02 13:35:06,026	autodl-tmp	INFO	[2.7380728721618652, 3.233557939529419, 27.444753646850586, 29.269224166870117, 3.1413462162017822, 5.771807670593262, 0, 0.00029985002812265633]
2025-12-02 13:35:09,221	autodl-tmp	INFO	Train Epoch: 6 [0%]
2025-12-02 13:35:09,221	autodl-tmp	INFO	[2.548027992248535, 2.199531316757202, 14.139811515808105, 23.331769943237305, 3.118525505065918, 5.878347396850586, 0, 0.000299812546869141]
2025-12-02 13:35:11,328	autodl-tmp	INFO	Train Epoch: 7 [0%]
2025-12-02 13:35:11,328	autodl-tmp	INFO	[2.8041880130767822, 2.0488972663879395, 23.078075408935547, 25.282947540283203, 2.9064574241638184, 5.081879615783691, 0, 0.00029977507030078235]
2025-12-02 13:35:13,458	autodl-tmp	INFO	Train Epoch: 8 [0%]
2025-12-02 13:35:13,459	autodl-tmp	INFO	[2.477522611618042, 2.765768527984619, 18.10857582092285, 23.826641082763672, 2.780367374420166, 4.596684455871582, 0, 0.00029973759841699473]
2025-12-02 13:35:14,520	autodl-tmp	INFO	Train Epoch: 9 [0%]
2025-12-02 13:35:14,520	autodl-tmp	INFO	[2.932689905166626, 2.3824915885925293, 23.709260940551758, 25.153249740600586, 2.74429988861084, 4.404598236083984, 0, 0.00029970013121719257]
2025-12-02 13:35:16,725	autodl-tmp	INFO	Train Epoch: 10 [0%]
2025-12-02 13:35:16,725	autodl-tmp	INFO	[2.650240898132324, 3.1976215839385986, 29.899538040161133, 33.03239440917969, 2.7884421348571777, 4.480808258056641, 0, ### 0.0002996626687007904]`
